src/main/java/org/spongepowered/common/data/property/SpongePropertyRegistry.java

```diff
@@ -104,7 +105,7 @@ public void completeRegistration() {
     public Collection<Property<?, ?>> getPropertiesFor(final PropertyHolder holder) {
         final ImmutableList.Builder<Property<?, ?>> builder = ImmutableList.builder();
         if (this.tempRegistry != null) { // Still doing registrations
-            final Set<Class<? extends Property<?, ?>>> used = new HashSet<>();
+            final Set<Class<? extends Property<?, ?>>> used = new ReferenceOpenHashSet<>();
```

Comment:
this is a moot set to even bother using. It's a temp registry that gets dropped as soon as the game has finished loading.

Reply:
Reverted with other temporary maps.